spinta/manifests/xsd2/helpers.py

```diff
@@ -481,7 +481,7 @@ def process_element(self, node: _Element, state: State, is_array=False, is_root=
 
         # ref - a reference to separately defined element
         if node.attrib.get("ref"):
-            property_name = node.attrib["ref"]
+            property_name = node.attrib["ref"].removeprefix("tns:")
```

Comment:
Remove only if `xmlns:tns` matches `targetNamespace`:

https://www.w3.org/TR/xmlschema-1/#src-resolve
https://stackoverflow.com/questions/17295588/xmlntns-and-targetnamespace

Reply:
Man pasirode, kad cia specifinis atvejis, kai `xmlns:tns` matches `targetNamespace`?